### PR TITLE
Reader: enable follow by url from sites search

### PR DESCRIPTION
**Pull request by samouri**
Addresses: https://github.com/Automattic/wp-calypso/issues/16594

Image:

### CODE REVIEW

**Comment by samouri** [comment]
question:  on following-manage we have a separator line.  do we want one here as well?
cc @fraying

**Comment by fraying** [comment]
Damn, @samouri, that was some crazy quick work.

I think this works perfectly. And the light line is a nice consistency touch. Well done!

**Comment by blowery** [comment]
rebased to pick up fix for tests

File: client/reader/search-stream/index.jsx [hunk 1]
```diff
@@ -180,6 +188,15 @@ class SearchStream extends React.Component {
 								</ControlItem>
 							</SegmentedControl> }
 					</CompactCard>
+					{ showFollowByUrl &&
+						<div className="search-stream__url-follow">
+							<FollowButton
+								followLabel={ translate( 'Follow %s', { args: queryWithoutProtocol } ) }
+								followingLabel={ translate( 'Following %s', { args: queryWithoutProtocol } ) }
+								siteUrl={ addSchemeIfMissing( readerAliasedFollowFeedUrl, 'http' ) }
+								followSource={ READER_FOLLOWING_MANAGE_URL_INPUT }
```
Comment (blowery) [inline client/reader/search-stream/index.jsx]: prolly want a new following source

Comment (samouri) [inline client/reader/search-stream/index.jsx]: good call! added

Comment (samouri) [inline client/reader/search-stream/index.jsx]: oh, you beat me to it

**Comment by samouri** [comment]
🚢

File: client/reader/search-stream/index.jsx (hunk 1 above)
Comment (blowery) [inline client/reader/search-stream/index.jsx]: added it

**Comment by blowery** [comment]
i'll ship this once tests go

**Comment by fraying** [comment]
Well done, y'all! Thanks.

**Pull request merged**